changelogs/fragments/copy-sanitize-check-mode-invocation-args.yaml

```diff
@@ -0,0 +1,7 @@
+bugfixes:
```

Comment:
@s-hertel Should this use the new `security_fixes` category that was backported?